src/uct/api/uct.h

```diff
@@ -1159,14 +1174,39 @@ UCT_INLINE_API ssize_t uct_ep_put_bcopy(uct_ep_h ep, uct_pack_callback_t pack_cb
 
 /**
  * @ingroup UCT_RMA
- * @brief
- */
-UCT_INLINE_API ucs_status_t uct_ep_put_zcopy(uct_ep_h ep, const void *buffer, size_t length,
-                                             uct_mem_h memh, uint64_t remote_addr,
-                                             uct_rkey_t rkey, uct_completion_t *comp)
+ * @brief Provides RDMA PUT operation
+ *
+ * The input data in @a iov array of @ref ::uct_iov_t structures sent to remote
+ * address ("gather output"). Buffers in @a iov are processed in array order.
+ * This means that the function complete iov[0] before proceeding to
+ * iov[1], and so on.
+ *
+ *
+ * @param [in] ep          Destination endpoint handle.
+ * @param [in] iov         Points to an array of @ref ::uct_iov_t structures.
+ *                         The @a iov pointer must be valid address of an array
+ *                         of @ref ::uct_iov_t structures. A particular structure
+ *                         pointer must be valid address. NULL terminated pointer
+ *                         is not required.
+ * @param [in] size        Size of the @a iov data @ref ::uct_iov_t structures
+ *                         array. If @a size is zero, no data operation provided.
+ *                         @a size is limited by @ref uct_iface_attr_cap_put_max_iov
```

Comment:
@yosefe 
'size' limited by uct_iface_attr::put::max_iov.
Total length of the iov data is limited by uct_iface_attr::put::max_zcopy (size * particular length in iov)
Am I right?